appveyor.yml

```diff
@@ -0,0 +1,55 @@
+
+os: Visual Studio 2015
+
+environment:
+  matrix:
+    - ARCH:                x64
+      D_COMPILER:          dmd
+      D_VERSION:           2.077.1
+      BRANCH:              master
+
+cache:
+
+init:
+  - git config --global core.autocrlf input
+
+build_script:
+  - cd c:/projects/
+  - ps: |
+    Start-FileDownload "http://downloads.dlang.org/releases/2.x/$Env:D_VERSION}/dmd.$Env:D_VERSION.windows.7z" -FileName dmd2.7z
```

Comment:
BTW instead of copy/pasting this all over, we are [working towards a windows variant of the installer script](https://github.com/dlang/installer/pull/275/files). It got a bit stalled though as I'm no expert on Windows nor user.

Reply:
> BTW instead of copy/pasting this all over, we are working towards a windows variant of the installer script. It got a bit stalled though as I'm no expert on Windows nor user.

I also noticed after creating this PR. Downloading the compiler is usually a smaller problem, but tedious.